### PR TITLE
add filter class for function calls

### PR DESCRIPTION
**Pull request by maur1**
## Description
Add function calls filter class

### Why:
None is read as filter input and the functions.list_calls() endpoint will not return any function calls. 
functions.calls.list(function_id=xxx) now returns a list of calls for the specified function id

### Still a bug to be solved:
Looks like the endpoint returns an empty list if any filter parameters are present

### CODE REVIEW

**Comment by erlendvollset** [comment]
Please also add a test which reproduces the issue.

**Comment by pavelzubarev** [comment]
@erlendvollset ;)

**Comment by erlendvollset** [comment]
You will need to bump version and update changelog aswell.

**Review by erlendvollset — APPROVED** [review]
Lgtm!

**Review by erlendvollset — CHANGES_REQUESTED** [review]
Little too quick

File: pyproject.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ pandas = { version = "*", optional = true }
 geopandas = { version = ">=0.10.0", optional = true }
 shapely = { version = ">=1.7.0", optional = true }
 pip = { version = ">=20.0.0", optional = true}
+coverage = "^6.5.0"
```
Comment (erlendvollset) [inline pyproject.toml:24]: This should be added to dev dependencies.

Comment (erlendvollset) [reply pyproject.toml:24]: Or why do we need to add it at all? pytest-cov should cover our needs.

**Review by erlendvollset — CHANGES_REQUESTED** [review]
Little too quick

**Pull request merged**